experimental/add_to_app/ios_fullscreen/IOSFullScreen.xcodeproj/project.pbxproj

```diff
@@ -494,7 +494,7 @@
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				COPY_PHASE_STRIP = NO;
 				DEBUG_INFORMATION_FORMAT = dwarf;
-				ENABLE_BITCODE = NO;
+				ENABLE_BITCODE = YES;
```

Comment:
YES is the default, so you can delete the `ENABLE_BITCODE` lines instead of setting to YES.

Reply:
Done.